packages/bind/src/applyBindings.ts

```diff
@@ -42,42 +55,48 @@ const bindingDoesNotRecurseIntoElementTypes = {
   'template': true
 }
 
-function getBindingProvider () {
+function getBindingProvider() {
   return options.bindingProviderInstance.instance || options.bindingProviderInstance
 }
 
-function isProviderForNode (provider, node) {
+function isProviderForNode(provider, node: HTMLElement): boolean {
```

Comment:
Node type 3 = Text / 8 = comment, change HTMLElement to Node. Please use base classes if it's possible.